docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
+services:
+  app:
+    container_name: resource-manager-app
+    image: resource-manager-app
+    build: ./
+    networks:
+      - resource-manager-net
+    ports:
+      - "8080:8080"
+    environment:
+      DB_HOST: postgres-db
+      POSTGRES_URL: jdbc:postgresql://postgres-db:5432/resource_manager_app_db
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres1209
```

Comment:
tutaj sam definiujesz haslo i to jest, ale pamietaj zeby nie przechowywac hasel w repozytorium np. jakbys konfigurowal poczte email czy cos - to tak na przyszlosc